src/murmur/Server.h

```diff
@@ -164,6 +165,7 @@ class Server : public QThread {
 	Timer tUptime;
 
 	bool bValid;
+	bool bUdp;
```

Comment:
```suggestion
	bool m_udp;
```
What is this variable used for? I think it could probably use a more descriptive name :thinking:

Reply:
Remember #4819? I was afraid that even if the server failed to bind to UDP port but it still negotiates UDP ciphers, so I added an explicit flag here.

Reply:
What would be the implications of such a scenario? I would assume that if the server doesn't connect to the UDP port anyways, there is no harm done by doing a cipher negotiation, is there?

Besides: If the server fails to bind to the UDP port, we should probably terminate the server anyways...